site2/docs/admin-api-packages.md

```diff
@@ -109,142 +124,168 @@ bin/pulsar-admin packages download functions://public/default/example@v0.1 --pat
+bin/pulsar-admin packages list-versions type://tenant/namespace/packageName
 ```
 
 <!--REST API-->
 
-{@inject: endpoint|PUT|/admin/v3/packages/:type/:tenant/:namespace/:packageName/:version/metadata/?version=[[pulsar:version_number]]}
+{@inject: endpoint|GET|/admin/v3/packages/:type/:tenant/:namespace/:packageName}
 
 <!--JAVA-->
-Update a package metadata information synchronously.
+List all versions of a package synchronously.
 
 ```java
-   void updateMetadata(String packageName, PackageMetadata metadata) throws PulsarAdminException;
+  List<String> listPackageVersions(String packageName) throws PulsarAdminException;
 ```
-Update a package metadata information asynchronously.
+
+List all versions of a package asynchronously.
 ```java
-   CompletableFuture<Void> updateMetadataAsync(String packageName, PackageMetadata metadata);
+  CompletableFuture<List<String>> listPackageVersionsAsync(String packageName);
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
 
-### Delete a specified package
-You can delete a specified package with its package name in the following ways.
+### List all packages of a specific type under a namespace
+
+You can use the following commands to list all packages of a specific type under a namespace.
 
 <!--DOCUSAURUS_CODE_TABS-->
-<!--pulsar-admin-->
-The following command example deletes a package of version 0.1.
 
+<!--pulsar-admin-->
 ```shell
-bin/pulsar-admin packages delete functions://public/default/example@v0.1
+bin/pulsar-admin packages list --type function public/default
 ```
 
 <!--REST API-->
 
-{@inject: endpoint|DELETE|/admin/v3/packages/:type/:tenant/:namespace/:packageName/:version/?version=[[pulsar:version_number]]}
+{@inject: endpoint|PUT|/admin/v3/packages/:type/:tenant/:namespace}
 
 <!--JAVA-->
-Delete a specified package synchronously.
+List all packages of a specific type under a namespace synchronously.
 
 ```java
-   void delete(String packageName) throws PulsarAdminException;
+  List<String> listPackages(String type, String namespace) throws PulsarAdminException;
 ```
-Delete a specified package asynchronously.
+
+List all packages of a specific type under a namespace asynchronously.
+
 ```java
-   CompletableFuture<Void> deleteAsync(String packageName);
+  CompletableFuture<List<String>> listPackagesAsync(String type, String namespace);
 ```
 <!--END_DOCUSAURUS_CODE_TABS-->
+
+
+> **Note**  
+> This page only includes some of the most frequently used operations.
+> * For more information about the commands, flags, descriptions, and more in Pulsar administrator, see [Pulsar admin](https://pulsar.apache.org/tools/pulsar-admin/) page.
+> * For more information about the parameters, responses, samples, and more in REST API, see [REST API](https://pulsar.incubator.apache.org/admin-rest-api#/) page.
+> * For more information about the classes, methods, descriptions, and more Java administrator API, see [Java admin API](https://pulsar.apache.org/api/admin/2.8.0-SNAPSHOT/) page.
```

Comment:
Can we keep it consistent across the guide? (move this to the beginning of the chapter)

Reply:
I don't think it's appropriate to put Note at the beginning of a page before you presenting anything.

Reply:
The reasons for moving `Important` to the beginning:

![image](https://user-images.githubusercontent.com/50226895/145557739-6447c48a-b70e-49df-baa5-08f57da85732.png)

- These are several chapters in the `Admin API` guide, each chapter has its operations.
However, in these chapters (except `Overview`), only frequently used operations (especially their flags and descriptions) are explained here since more detailed info is added directly to code files to generate various API docs automatically. This is beneficial for both developers/writers (contribute docs conveniently) and users (they are accustomed to go to the corresponding API pages which show the latest info).

![image](https://user-images.githubusercontent.com/50226895/145558128-e55aa9de-17db-481f-a0db-38d32fa2e159.png)

- Keep consistent with other chapters.
As you can see, this `Important` is added to every chapter at the beginning to remind users to go to API websites (where host latest and complete info) directly, which reduces the chance that users do not find desired info after they read through the whole chapter.